tutorials/FITS-images/FITS-images.ipynb

```diff
@@ -40,15 +58,33 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "QxeH6boSgEIp"
+   },
    "source": [
-    "Download the example FITS files for this tutorial."
+    "Download the example FITS files (which contain the image data for the Horsehead Nebula) for this tutorial.\n",
+    "\n",
+    "First, we need to download the FITS file image for this tutorial. This is an astronomical image of the Horsehead Nebula taken with a photographic plate. The image has been digitized, that is, the image has been scanned by a computer and converted to a 2-D array. Each position in the array corresponds with the projected position on the sky, and bright areas of the image have high values and dark areas have low values in the array. \n",
+    "\n",
+    "Images taken with astronomical instruments called [CCDs or ``charge-coupled devices''](https://en.wikipedia.org/wiki/Charge-coupled_device) are organized similarly. When illuminated by light, CCDs accumulate electrons, converting brightness values to electron counts. A CCD image is essentially a 2-D array, where each position on the array represents a single CCD pixel, and the values in that array represent the number of counts registered in that pixel."
```

Comment:
The quotes around `charge-coupled devices` don't work because the backticks have a special meaning in markdown: they indicate a chunk of literal text. Suggested change is below.
```suggestion
    "Images taken with astronomical instruments called [CCDs or "charge-coupled devices"](https://en.wikipedia.org/wiki/Charge-coupled_device) are organized similarly. When illuminated by light, CCDs accumulate electrons, converting brightness values to electron counts. A CCD image is essentially a 2-D array, where each position on the array represents a single CCD pixel, and the values in that array represent the number of counts registered in that pixel."
```